docs/source/llms/chat-model-guide/index.rst

```diff
@@ -0,0 +1,802 @@
+   - **Temperature and Max Tokens**: Controls the generation variability (``temperature``) and token limit for responses.
+
+2. **General Configuration**: Additional settings for the overall behavior of the model, such as how user responses should be framed for subsequent agents.
+
+Here's how we set the configuration for our agents:
+
+.. code-block:: python
+
+   model_config = {
+       "models": {
+           "judge": {
+               "endpoint": "databricks-meta-llama-3-1-405b-instruct",
+               "instruction": (
+                   "You are an evaluator of answers provided by others. Based on the context of both the question and the answer, "
+                   "provide a corrected answer if it is incorrect; otherwise, enhance the answer with additional context and explanation."
+               ),
+               "temperature": 0.5,
+               "max_tokens": 2000,
+           },
+           "oracle": {
+               "endpoint": "databricks-mixtral-8x7b-instruct",
+               "instruction": (
+                   "You are a knowledgeable source of information that excels at providing detailed, but brief answers to questions. "
+                   "Provide an answer to the question based on the information provided."
+               ),
+               "temperature": 0.9,
+               "max_tokens": 5000,
+           },
+       },
+       "configuration": {
+           "user_response_instruction": "Can you evaluate and enhance this answer with the provided contextual history?"
+       },
+   }
+
+Benefits of External Configuration
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+- **Flexibility**: The decoupled configuration allows us to easily switch or adjust model behavior without modifying the core logic. For example, we can 
+  change the model's instructions or adjust the ``temperature`` to test different levels of creativity in the responses.
+  
+- **Scalability**: As more agents are added to the system or new roles are introduced, we can extend this configuration without cluttering the model's 
+  code. This separation keeps the codebase cleaner and more maintainable.
+
+- **Reproducibility and Comparison**: By keeping configuration external, we can log the specific settings used in each run with MLflow. This makes it 
+  easier to reproduce results and compare different experiments, ensuring a robust evaluation and adjudication process to select the best performing
+  configuration.
+
+With the configuration in place, we're now ready to log the model and run experiments using these settings. By leveraging MLflow's powerful tracking 
+and logging features, we'll be able to manage the experiments efficiently and extract valuable insights from the agent's responses.
+
+Defining an Input Example
+-------------------------
+
+Before logging our model, it's important to provide an ``input_example`` that demonstrates how to interact with the model. This example serves several key purposes:
```

Comment:
If users don't provide one, ChatModel will set a default input example https://github.com/mlflow/mlflow/blob/master/mlflow/types/llm.py#L459, can we also mention it here just in case user see an example even they don't set it

Reply:
great callout! Added a note